web_src/js/modules/tippy.js

```diff
@@ -55,6 +57,15 @@ function attachTooltip(target, content = null) {
   return target._tippy;
 }
 
+function switchTitleToTooltip(target) {
+  const title = target.getAttribute('title');
+  if (title) {
+    target.setAttribute('data-tooltip-content', title);
+    target.setAttribute('aria-label', title);
+    target.setAttribute('title', '');
```

Comment:
```suggestion
    target.removeAttribute('title');
```

Reply:
It is kept here by purpose. See new comment

Reply:
I think removing the attribute may trigger the infinite loop because the time element then tries to set it, we delete it, it sets again, ad ininitum...